CONTRIBUTING.md

```diff
@@ -4,7 +4,28 @@
 1. All python code must be python 2.6/2.7/3.6 compatible
 1. The code should follow linting from pylint
 
-## Unit tests
+## Developing locally
+
+Dependencies: 
+https://virtualenv.pypa.io/en/latest/installation.html
```

Comment:
```suggestion
```
I don't think mentioning this is important.

Reply:
ok, fixed